README.md

```diff
@@ -1931,6 +1972,117 @@ port-channel hash input offset. Valid values are integer or 'default'. This prop
 port-channel symmetry hash. Valid values are true, false or 'default'. This property is only supported on N9k, N30xx, N31xx.
 
 --
+### Type: cisco_stp_global
+Manages configuration of a spanning tree global parameters
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I2(1)        | 1.3.0                  |
+| N30xx    | 7.0(3)I2(1)        | 1.3.0                  |
+| N31xx    | 7.0(3)I2(1)        | 1.3.0                  |
+| N56xx    | 7.3(0)N1(1)        | 1.3.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.3.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.3.0                  |
+| IOS XR   | unsupported        | unsupported            |
+
+#### <a name="cisco_stp_global-caveats">Caveats</a>
+
+| Property | Caveat Description |
+|:--------|:-------------|
+| `bd_designated_priority` <br> `bd_forward_time` <br> `bd_hello_time` <br> `bd_max_age` <br> `bd_priority` <br> `bd_root_priority` | Supported only on N7k |
+| `domain` | Supported only on N56k, N6k, N7k |
+| `fcoe` | Supported only on N9k, N30xx, N31xx |
+
+#### Parameters
+
+##### `bd_designated_priority`
+designated bridge priority. Valid values are an array of [bd_range, designated_priority] pairs or 'default'. This property is supported only on N7k.
```

Comment:
I think it would be ok to omit the `This property is supported only on N7K.` bit since it is redundant with the `Caveats` table.